src/views/Completed.vue

```diff
@@ -209,6 +198,64 @@ export default defineComponent({
       });
       return popover.present();
     },
+    async getCompletedOrders(vRow?: any, vStart?: any) {
+      vRow = vRow ? vRow : process.env.VUE_APP_VIEW_SIZE;
+      vStart = vStart ? vStart : 0;
+      const payload = {
+        "json": {
+          "params": {
+            "start": vStart,
+            "rows": vRow,
+            "sort": "reservedDatetime desc",
+            "group": true,
+            "group.field": "orderId",
+            "group.limit": 1000,
+            "group.ngroups": true,
+            "defType": "edismax",
+            "q.op": "AND",
+            "qf": "orderId"
+          },
+          "query": "(* *)",
+          "filter": ["docType:ORDER","orderTypeId: SALES_ORDER","orderStatusId:ORDER_COMPLETED","-shipmentMethodTypeId : STOREPICKUP",`facilityId: ${this.currentFacility.facilityId}`],
+          "facet": {
+            "shipmentMethodTypeIdFacet": {
+              "excludeTags": "shipmentMethodTypeIdFilter",
+              "field": "shipmentMethodTypeId",
+              "mincount": 1,
+              "limit": -1,
+              "sort": "index",
+              "type": "terms",
+              "facet": {
+                "ordersCount": "unique(orderId)"
+              }
+            }
+          }
+        }
+      }
+      this.store.dispatch("orders/getCompletedOrders", payload)
+    },
+    async loadMoreOrders(event: any) {
+      this.getCompletedOrders(process.env.VUE_APP_VIEW_SIZE, Math.ceil(Object.getOwnPropertyNames(this.orders).length/process.env.VUE_APP_VIEW_SIZE)*10)
+        .then(()=> {
+          event.target.complete();
+        })
+    }
+  },
+  mounted(){
```

Comment:
```suggestion
  mounted() {
```

Reply:
Sorry Sir, that was left by mistake, I have added a space where you suggested to me.